addons/website_slides/models/res_partner.py

```diff
@@ -59,13 +73,15 @@ def _compute_slide_channel_company_count(self):
     def action_view_courses(self):
         """ View partners courses. In singleton mode, return courses followed
         by all its contacts (if company) or by themselves (if not a company).
-        Otherwise simply set a domain on required partners. """
+        Otherwise simply set a domain on required partners. The courses to which
+        the partner(s) is not enrolled (e.g. invited) are not shown. """
         action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
-        action['name'] = _('Followed Courses')
+        action['display_name'] = _('Followed Courses')
```

Comment:
Side remark: "followed" is a bit strange, isn't it frenglish ?

Reply:
Indeed, seems like frenglish. _Taken_ Courses? (Followed It is used in a few views, and in a security rule in website_slides_forum).